docs/introduction/project-setup.mdx

```diff
@@ -150,36 +150,35 @@ patch or create if they are missing:
 Open your main build script and apply the suggested changes:
 
 ```diff title="android/build.gradle"
- buildscript {
+buildscript {
    ext {
-     buildToolsVersion = "31.0.0"
+     buildToolsVersion = "33.0.0"
 // highlight-next-line
      minSdkVersion = 21 // (1)
-     compileSdkVersion = 30
-     targetSdkVersion = 30
+     compileSdkVersion = 33
+     targetSdkVersion = 33
 // highlight-next-line
-+    kotlinVersion = 'X.Y.Z' // (2)
++    kotlinVersion = "X.Y.Z" // (2)
+     // We use NDK 23 which has both M1 support and is the side-by-side NDK version from AGP.
+      ndkVersion = "23.1.7779620"
```

Comment:
Isn't the version too specific? I guess just after 2-4 weeks the build number `7779620` will be different when someone installs the latest 23.x NDK. @d4vidi , @xiongemi please advise.